packages/python/accern_data/accern_data.py

```diff
@@ -287,6 +291,13 @@ def _write_cols(self) -> None:
     def add_result(self, signal: pd.DataFrame) -> None:
         fname = self.get_path(is_by_day=self._is_by_day)
         tmp_fname = get_tmp_file_name(fname)
+
+        def micro_to_milli(timestamp: pd.Timestamp) -> str:
+            return micro_to_millisecond(
+                pd.to_datetime(timestamp).strftime(DATETIME_FORMAT))
+
+        for col in ["harvested_at", "crawled_at", "published_at"]:
+            signal.loc[:, col] = signal.loc[:, col].apply(micro_to_milli)
```

Comment:
weird that the SettingWithCopyWarning doesn't appear here

Reply:
that should be removed. idk why i kept this